apps/src/applab/applab.js

```diff
@@ -1717,14 +1719,17 @@ Applab.createSharedRecord = function (opts) {
   var record = codegen.marshalInterpreterToNative(Applab.interpreter,
```

Comment:
I don't think this marshalInterpreterToNative() call is needed anymore. The parameters to any functions that you expose to the interpreter should be marshaled already.